docs/development.md

```diff
@@ -188,6 +189,23 @@ KubeFed controller manager:
 
 Follow the [cleanup instructions in the user guide](../charts/kubefed/README.md#uninstalling-the-chart).
 
+## Embedding static files using go-bindata
+This project is using `go-bindata` tool for embedding static files into its e2e test-suite to enable the creation of a self-contained e2e binary.
+
+You can install this utility using the following command:
```

Comment:
Fixed.

Reply:
Now that go-bindata is downloaded by the script, please update this documentation to reference the need to execute the script rather than indicating how to install the dependency directly.